llvm-node.d.ts

```diff
@@ -190,15 +190,18 @@ declare namespace llvm {
   }
 
   class ConstantInt extends Constant {
-    static get(context: LLVMContext, value: number, numBits?: number, signed?: boolean): ConstantInt;
+    static get(context: LLVMContext, value: number|string, numBits?: number, signed?: boolean): ConstantInt;
 
     static getFalse(context: LLVMContext): ConstantInt;
 
     static getTrue(context: LLVMContext): ConstantInt;
 
     private constructor();
 
-    readonly value: number;
+    /**
+     * Return string for integers with more then 54 bits
+     */
+    readonly value: number|string;
```

Comment:
This is unfortunately a breaking change. Therefore, it can only be released with another major version. 
I also believe that it makes the API quite cumbersome. A user always needs to check if it is a number or a string. 

I think we should investigate some time in finding out how an API using big int could look like and what the drawbacks are (e.g. only supported in node 10+). An other alternative is to expose the string operations using different methods / getters.

Reply:
For now, I added `getValueAsString()` method, for `v3` we need to bump minimal version of Node and return BigInt :)